### ISSUE TITLE
Account sorting is forced alphabetical

### ISSUE THREAD

**Issue by xagotrunk** [Jessecar96/SteamDesktopAuthenticator#610]
The new update to 1.0.10 forces all the accounts to be sorted alphabetically in the app window rather than the order they are in the manifest.json. Is there a way to change this?

**Comment by ChargedByte** [comment]
Currently, no I can take a look this later

**Comment by wazerstar** [comment]
@ChargedByte 
@Jessecar96 

Yes this latest release makes ctrl + up/down no longer working it worked in past update, and was introduced in Version 1.0, after personally suggesting it.


```

Version 1.0 - First stable-ish release

@Jessecar96 Jessecar96 released this on Dec 23, 2015
```

**Comment by playeremers** [comment]
any plans to implement this broken feature that @wazerstar talked about?  i didn't even know it was a feature to sort by using ctrl up/down
i really need this, have many accounts and find kinda anoying to deal with the way SDA automatically sorted it 
thanks
